src/lib/y2storage/dasd.rb

```diff
@@ -0,0 +1,50 @@
+# encoding: utf-8
+
+# Copyright (c) [2017] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "y2storage/storage_class_wrapper"
+require "y2storage/partitionable"
+
+module Y2Storage
+  # A DASD (direct-access storage device), typically used in mainframes
```

Comment:
Other than a Wikipedia article with little useful content, there is not much; at IBM web pages, there are various articles that touch the fringes of the topic, but I have not found anything yet that could be called a useful reference documentation or a useful tutorial. It is not the task of a class documentation to magically conjure up documentation that is neglected and absent everywhere else.